ostree_upload_server/bundle_importer.py

```diff
@@ -43,7 +43,7 @@ def import_bundle(bundle, repository, gpg_homedir=None, keyring=None,
         sign_key = sign_key or defaults.get('sign_key', None)
 
         logging.info("Starting the bundle import process...")
-        for arg in inspect.getargspec(BundleImporter.import_bundle)[0]:
+        for arg in inspect.getfullargspec(BundleImporter.import_bundle)[0]:
```

Comment:
This is a pretty weird way to log what is rather a short list of parameters.

Reply:
Indeed. I considered tearing it out, but opted to leave it working the way it is.